### PR TITLE
fix(ListingCarousel): stops wrong listing cell from being activated o…

### PR DESCRIPTION
**Pull request by yasabere**
…n repeated click

affects: @rentpath/react-ui-ag

- Added debounce to on click function

### CODE REVIEW

File: packages/react-ui-ag/src/Carousels/ListingCarousel.js [hunk 1]
```diff
@@ -30,7 +34,18 @@ export default class ListingCarousel extends PureComponent {
 
   constructor(props) {
     super(props)
-    this.state = { selectedIndex: this.props.selectedIndex }
+    this.state = {
+      selectedIndex: this.props.selectedIndex,
+      activelySliding: false,
+    }
```
Comment (bethstevic) [inline packages/react-ui-ag/src/Carousels/ListingCarousel.js]: I don't see where you are using or setting `this.state.activelySliding`?

Comment (yasabere) [reply packages/react-ui-ag/src/Carousels/ListingCarousel.js]: Fixed

**Comment by bethstevic** [comment]
Looks good :+1:

File: packages/react-ui-ag/src/Carousels/__tests__/ListingCarousel-test.js [hunk 2]
```diff
@@ -228,7 +233,20 @@ describe('ListingCarousel', () => {
     const wrapper = mount(<ListingCarousel {...props} onSlide={onSlide} />)
     expect(onSlide).not.toHaveBeenCalled()
     wrapper.find(MobileMapListing).at(1).simulate('click')
-    expect(onSlide).toHaveBeenCalled()
+    setTimeout(() => {
+      expect(onSlide).toHaveBeenCalled()
+    }, DEBOUNCE_WAIT)
+  })
+
+  it('waits half a second to change slected index on click', () => {
```
Comment (nray621) [inline packages/react-ui-ag/src/Carousels/__tests__/ListingCarousel-test.js]: nitpick: typo

Comment (yasabere) [reply packages/react-ui-ag/src/Carousels/__tests__/ListingCarousel-test.js]: Rage code typo fixed

**Comment by nray621** [comment]
LGTM 👍

**Pull request merged**